
Mon, 27 Dec 2010 19:44:56 GMT

COMMAND LINE:
/usr/local/src/pypy-1.4.1-linux64/bin/pypy knucleotide.pypy-8.pypy 0 < knucleotide-input250000.txt

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "app_main.py", line 53, in run_toplevel
  File "knucleotide.pypy-8.pypy", line 12, in <module>
    from multiprocessing import Pool
ImportError: No module named multiprocessing
